api/swagger-spec/swagger.yml

```diff
@@ -275,6 +301,13 @@ definitions:
         type: array
         items:
           $ref: "#/definitions/componentVersion"
+  doctorInfo:
+    type: object
+    required:
+      - cluster
+    properties:
+      cluster:
+        $ref: "#/definitions/cluster"
```

Comment:
we can get uuid from the url params. so avoided here . if you want me to add it here then URL has to change to just /doctor